.gitignore

```
@@ -1,3 +1,4 @@
 .cache
 *.swp
-__pycache__*
\ No newline at end of file
+__pycache__*
+.vscode
```

```suggestion
.vscode
```
Missing `newline` at the end of the file